acceptance-tests/features/us31_view_case_list.feature

```diff
@@ -11,14 +11,14 @@ Acceptance Criteria:
   - Last updated date
 
 Background:
-  Given I know how many items are originally in the list
-  Given I have added values to the case_api
+  Given I view the case list
```

Comment:
should this not be the other way around, you want to add values to the case list before you view the case list